model-optimizer/extensions/front/tf/mvn.py

```diff
@@ -88,15 +89,13 @@ def infer(node: Node):
             log.warning('Reduction indices for mean and variance for MVN node {} are not constants'.format(node.name))
             return
 
-        if not (all(node.in_node(1).value == node.required_reduction_indices) and
-                    all(node.in_node(2).value == node.required_reduction_indices)):
-            log.warning('Reduction indices for mean {} and variance {} do not match required ones {}'.format(
+        if not (all(node.in_node(1).value == node.in_node(2).value)):
+            log.warning('Reduction indices for mean {} and variance {} do not match'.format(
                 node.in_node(1).value,
-                node.in_node(2).value,
-                node.required_reduction_indices
+                node.in_node(2).value
             ))
             return
 
         node.graph.remove_edge(node.in_node(2).id, node.id)
```

Comment:
Why the input port 1 is disconnected and the input port 2 is kept? So we will have MVN with connected port 0 and 2 which is wrong.

Reply:
Please, add a unit test to break your solution.

Reply:
I dont see how this code is wrong, it removes edge with id=2, and in the end, we have MVN with edges 0 and 1. But the code still will be refactored to use ports.

Reply:
Sorry for that. I incorrectly interpreted the code highlighting.